ci-operator/jobs/redhat-developer/odo/redhat-developer-odo-main-periodics.yaml

```diff
@@ -340,18 +340,10 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: aws
     ci-operator.openshift.io/cloud-cluster-profile: aws
-    ci-operator.openshift.io/variant: v4.9
+    ci-operator.openshift.io/variant: v4.13
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-redhat-developer-odo-main-v4.9-integration-e2e-periodic
-  reporter_config:
```

Comment:
It looks like there is no reporter_config for 4.13.  Does this mean that we won't get slack notifications?

Reply:
Updated!